docs/distributed-tracing-guide.asciidoc

```diff
@@ -55,17 +55,38 @@ And should respond to it with these headers:
 
 [source,header]
 ----
-Access-Control-Allow-Headers: elastic-apm-traceparent
+Access-Control-Allow-Headers: traceparent
 Access-Control-Allow-Methods: [allowed-methods]
 Access-Control-Allow-Origin: [request-origin]
 ----
 
-NOTE: `Access-Control-Request-Headers` might include more headers than just `elastic-apm-traceparent`.
+NOTE: `Access-Control-Request-Headers` might include more headers than just `traceparent`.
 The response should include all headers if the server wishes to let the browser send the original request.
 
 NOTE: To make sure all components in a distributed trace are included,
 the sampling rate of backend agents might be affected by the sampling rate of the RUM agent.
 
+
+[float]
+[[backend-agents-compatibility]]
+=== Backend agents compatibility
+Since version 5.0, the RUM agent supports the official W3C tracecontext `traceparent` header,
+instead of the previously used `elastic-apm-traceparent` header.
+In this table you can find which versions of our backend agents also supports the official W3C 
+tracecontext header and therefore can be used with the RUM agent version >=5.0 for distributed tracing:
```

Comment:
```suggestion
[[backend-agent-compatibility]]
=== Backend Agent compatibility

Starting in version 5.0, the RUM Agent supports the official W3C tracecontext `traceparent` header,
instead of the previously used `elastic-apm-traceparent` header.
Use the table below to determine which versions of our backend agents also support the official W3C
tracecontext headers. Compatible agents use the official tracecontext spec to propagate traces and can
therefor be used with the RUM Agent version >=5.0 for distributed tracing.
```